pegasus/sites.v3/hourofcode.com/i18n/public/pt/thanks.md

```diff
@@ -30,32 +30,34 @@ social:
 <br /> [<button>Obtenha cartazes</button>](https://store.code.org/products/code-org-posters-set-of-12) Use o código de oferta FREEPOSTERS</p> 
 
-<br /> **A Hora do Código funciona durante <%= campaign_date('full') %>. Entraremos em contato sobre novos tutoriais e outras atualizações emocionantes à medida que surgirem. Enquanto isso, o que você pode fazer agora?**
+<% if @country == 'us' %> Thanks to the generosity of Ozobot, Dexter Industries, littleBits, and Wonder Workshop, over 100 classrooms will be selected to receive robots or circuits for their class! To be eligible to receive a set, make sure to complete the survey sent from Code.org after the Hour of Code. Code.org will select the winning classrooms. In the meantime, check out some of the robotics and circuits activities. Please note that this is only open for US schools. <% end %>
+
+<br /> **The Hour of Code runs during <%= campaign_date('full') %> and we'll be in touch about new tutorials and other exciting updates as they come out. In the meantime, what can you do now?**
 
 ## 1. Divulgue o evento em sua escola e comunidade
 
-Você acabou de se juntar ao movimento Hora do Código. Diga aos seus amigos com **#HourOfCode**!
+You just joined the Hour of Code movement. Tell your friends with **#HourOfCode**!
 
-<%= view :share_buttons, facebook:facebook, twitter:twitter %> <br /> Incentive os outros a participar [com nossos e-mails de amostra.](<%= resolve_url('/promote/resources#sample-emails') %>) Entre em contato com seu diretor e desafie todas as salas de aula da sua escola para se inscrever. Recrute um grupo local - clubes de escoteiros ou escoteiras, igrejas, universidades, sindicatos ou até mesmo alguns amigos. Você não precisa estar na escola para aprender novas habilidades. Convide um político local ou legislador para visitar sua escola para a Hora do Código. Isso pode ajudá-lo a conseguir suporte para o ensino da Ciência da Computação para além de uma hora.
+<%= view :share_buttons, facebook:facebook, twitter:twitter %> <br /> Encourage others to participate [with our sample emails.](<%= resolve_url('/promote/resources#sample-emails') %>) Contact your principal and challenge every classroom at your school to sign up. Recruit a local group — boy/girl scouts club, church, university, veterans group, labor union, or even some friends. Você não precisa estar na escola para aprender novas habilidades. Invite a local politician or policy maker to visit your school for the Hour of Code. Isso pode ajudá-lo a conseguir suporte para o ensino da Ciência da Computação para além de uma hora.
```

Comment:
Looks like Portuguese, Brazilian got cleared too. Obviously it's only affecting those that already had it translated, but not sure how many there are. So weird it's only "deleting" some of them though. I would think if it's some kind of sync setting that it would apply to the whole file. And it's not like this is driven by key names that were changed. I guess Crowdin just got affected by some of the other changes in the file and recognized these as new strings?